src/puppetlabs/puppetdb/cheshire.clj

```diff
@@ -19,6 +19,12 @@
   "Non-memoize version of add-common-json-encoders!"
   []
   (generate/add-encoder
+   org.postgresql.util.PGobject
+   (fn [data jsonGenerator]
+     (if (.getPrettyPrinter jsonGenerator)
```

Comment:
Yeah, I know... the reason I'm looking for it in particular here is because this is doing something interesting (.getPrettyPrinter) to the parameter, and I want to know where to go look for documentation.